test/c3/appframework/appframework_test.go

```diff
@@ -60,7 +60,7 @@ var _ = Describe("c3appfw test", func() {
 	})
 
 	Context("Single Site Indexer Cluster with Search Head Cluster (C3) and App Framework", func() {
-		It("smoke, c3, appframework: can deploy a C3 SVA with App Framework enabled, install apps then upgrade them", func() {
+		It("smoke, c3, appframework, appframeworksc3: can deploy a C3 SVA with App Framework enabled, install apps then upgrade them", func() {
```

Comment:
Shouldn't the `appframework` selector be removed? Same comment for others too

Reply:
If somewants to run all `appframework` test they will need that keyword for test selection.